app/page.tsx

```diff
@@ -67,21 +70,25 @@ function getBranchInfosFromExperiment(recipe: NimbusExperiment) : BranchInfo[] {
         branchInfo.id = value.content.tag;
         break;
 
+      case 'spotlight': 
+        branchInfo.id = value.id;
+        branchInfo.previewLink = getPreviewLink(_substituteLocalizations(value, recipe.localizations?.[Object.keys(recipe.localizations)[0]]));
+        break;
+
       case 'multi':
         // XXX only does first message
-	const firstMessage = value.messages[0]
+	    const firstMessage = value.messages[0]
         if (!('content' in firstMessage)) {
-	  console.warn('template "multi" first message does not contain content key; details not rendered')
-	  return branchInfo
+      	  console.warn('template "multi" first message does not contain content key; details not rendered')
+      	  return branchInfo
         }
 
-	// XXX only does first screen
+	      // XXX only does first screen
```

Comment:
Outdent to match the context, please.

Reply:
Looks like there was a weird combo of tabs and spaces here. Fixed!